cmds/modify-l1_cmds/transfer-proxy.js

```diff
@@ -0,0 +1,54 @@
+const ob = require('urbit-ob')
+const ajs = require('azimuth-js')
+const _ = require('lodash')
+const {files, validate, eth, azimuth, findPoints} = require('../../utils')
+
+exports.command = 'transfer-proxy'
+exports.desc = 'Set the transfer proxy of one or more points.'
+
+exports.builder = function(yargs) {
+}
+
+exports.handler = async function (argv)
+{
+  const workDir = files.ensureWorkDir(argv.workDir);
+  const privateKey = await eth.getPrivateKey(argv);
+  const ctx = await eth.createContext(argv);
+  const ethAccount = eth.getAccount(ctx.web3, privateKey);
+
+  const wallets = argv.useWalletFiles ? findPoints.getWallets(workDir) : null;
+  const points = findPoints.getPoints(argv, workDir, wallets);
+
+  console.log(`Will set transfer proxy for ${points.length} points`);
+  for (const p of points) 
+  {
+    let patp = ob.patp(p);
+    console.log(`Trying to set transfer proxy for ${patp} (${p}).`);
+
+    let wallet = argv.useWalletFiles ? wallets[patp] : null;
+    let targetAddress = 
+      argv.address != undefined
+      ? argv.address 
+      : argv.useWalletFiles 
+      ? wallet.ownership.keys.address :
```

Comment:
The wallet.json has `ownership.keys` and `transfer.keys` so I'm not sure which is the correct one to use here.

My assumption is that the goal is for the point to end up being owned by the ownership key, so I should set the transfer proxy to the ownership key and then the ownership key will use the transfer proxy rights to transfer the point to itself. But then I don't see what is the point of the separate "transfer" keys in the wallet.